matrixprofile/algorithms/snippets.py

```diff
@@ -105,6 +106,18 @@ def snippets(ts, snippet_size, num_snippets=2, window_size=None):
     # compute the fraction of each snippet
     for snippet in snippets:
         mask = (snippet['distance'] <= total_min)
+        # create a key "neighbors" for the snippet dict, 
+        # and store all the time series indices for the data represented by a snippet (arr[mask])
+        arr = np.arange(len(mask))
+        # max_index indicates the length of a profile, which is (n-m) in the Snippets paper)
+        max_index = time_series_len - snippet_size
```

Comment:
"n" is already defined which is the length of the time series.

Reply:
Thanks for your suggestion. However, I added a breakpoint to check variables, and it turns out that the value of n is 1 at this program point.
![debug](https://user-images.githubusercontent.com/32326618/90425821-3a2c5200-e075-11ea-8ea4-d9ce5584bde1.png)